src/content/quickstart/kind/_index.md

```diff
@@ -64,6 +64,14 @@ kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment n
 subctl export service --kubeconfig output/kubeconfigs/kind-config-cluster3 --namespace default nginx
 ```
 
+To verify a headless service follow the steps below instead of the above one
```

Comment:
```suggestion
##### Verify Headless Service
To verify deployment using a headless service use following steps
```
Similarly add a subsection for existing content with `##### Verify ClusterIP Service`